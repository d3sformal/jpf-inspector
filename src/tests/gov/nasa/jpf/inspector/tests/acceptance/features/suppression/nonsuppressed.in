# No suppression:
disable thread 1
en t 1

create breakpoint property_violated

run

finish
