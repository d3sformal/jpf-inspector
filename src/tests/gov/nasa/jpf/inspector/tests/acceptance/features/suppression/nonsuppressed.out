The thread 1 is now disabled.
The thread 1 is now enabled.
INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 property_violated]
INFO: JPF Terminating